gallery/src/styles/PageLayout/PageLayoutPage.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
+import RawHtmlExample from '../../CodeExample';
+
+const SingleColumnLayoutExampleCode = require('!!raw-loader!./SingleColumnLayoutExample.html').default,
+    TwoColumnLayoutExampleCode = require('!!raw-loader!./TwoColumnLayoutExample.html').default,
+    ReactRootLayoutExampleCode = require('!!raw-loader!./ReactRootLayoutExample.html').default;
+
+const PageLayoutGuidelinesPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p>
+        HTML and SCSS are provided for both single and two column page layouts. We use{' '}
+        <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox">
+          flexbox
+        </a>
+        to handle our layouts.
```

Comment:
This'll lose a space.
```suggestion
        </a>
        {' '}to handle our layouts.
```